test/colorado_lottery_test.rb

```diff
@@ -0,0 +1,165 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/contestant'
+require "./lib/game"
+require "./lib/colorado_lottery"
+
+class ColoradoLotteryTest < Minitest::Test
+  def test_it_exists
+    lottery = ColoradoLottery.new
+    assert_instance_of ColoradoLottery, lottery
+  end
+
+  def test_it_has_attributes
+    lottery = ColoradoLottery.new
+    assert_equal ({}), lottery.registered_contestants
+    assert_equal [], lottery.winners
+    assert_equal ({}), lottery.current_contestants
+  end
+
+  def test_contestant_interested_and_18
+    lottery = ColoradoLottery.new
+    pick_4 = Game.new('Pick 4', 2)
+    mega_millions = Game.new('Mega Millions', 5, true)
+    cash_5 = Game.new('Cash 5', 1)
+    winston = Contestant.new({
+                         first_name: 'Winston',
+                         last_name: 'Churchill',
+                         age: 18,
+                         state_of_residence: 'CO',
+                         spending_money: 5})
+    frederick = Contestant.new({
+                           first_name:  'Frederick',
+                           last_name: 'Douglas',
+                           age: 55,
+                           state_of_residence: 'NY',
+                           spending_money: 20})
+    benjamin = Contestant.new({
+                           first_name: 'Benjamin',
+                           last_name: 'Franklin',
+                           age: 17,
+                           state_of_residence: 'PA',
+                           spending_money: 100})
+    alexander = Contestant.new({
+                           first_name: 'Alexander',
+                           last_name: 'Aigades',
+                           age: 28,
+                           state_of_residence: 'CO',
+                           spending_money: 10})
+    alexander.add_game_interest('Pick 4')
+    alexander.add_game_interest('Mega Millions')
+    frederick.add_game_interest('Mega Millions')
+    winston.add_game_interest('Cash 5')
+    winston.add_game_interest('Mega Millions')
+    benjamin.add_game_interest('Mega Millions')
+    # assert_equal true, lottery.interested_and_18?(alexander, pick_4)
```

Comment:
you commented this assertion out (i assume because it's not passing), but it's especially important, because the way you have your method written, it can never return true. i definitely get the temptation in a timed setting, but don't take out parts of your test to make them pass-- it defeats their purpose 🙂